backend/public-graph/graph/resolver.go

```diff
@@ -2290,6 +2327,28 @@ func (r *Resolver) AddTrackProperties(ctx context.Context, sessionID int, events
 				return e.New("error deserializing track event properties")
 			}
 
+			attributes := make(map[string]string)
+			for k, v := range propertiesObject {
+				attributes[k] = fmt.Sprintf("%v", v)
+			}
+
+			// make event name the event key and delete from map
+			eventName := attributes["event"]
```

Comment:
what is the fallback name?

Reply:
Good call! I'll update to have a default, maybe "unknown_event", but wondering if we should log these cases since its not an ideal case